webpack.config.ts

```diff
@@ -21,6 +22,25 @@ module.exports = (_env: string, argv: Record<string, boolean | number | string>)
 	},
 	module: {
 		rules: [
+			{
+				test: /options-storage/,
+				loader: 'string-replace-loader',
+				options: {
+					search: '__featuresList__',
+					replace: (() => {
```

Comment:
This could be a function that runs every time the files change, but it's alright for now. If we find a better replacer webpack plugin the change will be _deleting 4 characters_